recipes/openscenegraph-osgQt/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "openscenegraph-osgQt" %}
+{% set version = "3.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openscenegraph/osgQt/archive/{{ version }}.tar.gz
+  sha256: dcc4436590639168e6470fe9c5343c82bca738d3296ebee014f40f2dc029afa1
+  patches:
+    - patches/osx.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkg-config
+    - doxygen
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
```

Comment:
I guess this is for openGL? If so, see https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

Reply:
@chrisburr Thank you for the precision, it will be usefull also for other package I have to build !